eras/babbage/impl/testlib/Test/Cardano/Ledger/Babbage/Imp/TxSpec.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Test.Cardano.Ledger.Babbage.Imp.TxSpec (
```

Comment:
Everything in ledger is related to a transaction, so it is not very good organizational approach. Convention that is being established is that test are focused on checking a specific rule. Test in this module seems to be mostly related to `UTXO` rule, so let's rename the module to 
```suggestion
module Test.Cardano.Ledger.Babbage.Imp.UtxoSpec (
```

Reply:
I will add the minFee specs in the same module, so I'll specialize it to RefScripts as `UtxoRefScriptSpec`